_rules/non-visual-reference-alternative-825ule.md

```diff
@@ -0,0 +1,481 @@
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 2
+
+The content in the second column is part of the navigation. The user is told to find the navigation on the right (which is a visual indicator word) and the navigation has a navigation element, but there are 2 navigation elements on the page so the user doesn't know which one to use.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>
+				<a href="https://www.w3.org/">W3C homepage</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/standards/">Standards</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/participate/">Participate</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/Consortium/membership">Membership</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+The content in the second column is indicated with the word "box" (which is a shape indicator based on visual perception) and is also indicated by referencing the word "howdy", but the word "howdy" is in a different paragraph than the word "box".
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the box on the right, for a surprise</p>
+			<p>If you can't find the box here is a hint: it says "Howdy"!</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 4
+
+This document is using the word "triangle" but in this case the triangle menu is on a different page.
```

Comment:
Mention that in the page with a triangle menu, the menu is not identified as a triangle menu.